web/components/admin/ApplicantInfo/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { firebaseConnect, firestoreConnect } from 'react-redux-firebase';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+import applicantCollections from '../../../util/applicantCollections';
+
+const mapPropsToQueries = (props) => {
+  const { applicantType, applicantId } = props;
+  return [
+    { collection: applicantCollections[applicantType].shortInfo, doc: applicantId },
+    { collection: applicantCollections[applicantType].longInfo, doc: applicantId },
+  ];
+};
+
+class ApplicantInfo extends React.Component {
+  getApplicantShortInfo = () => {
+    const { firestore, applicantId, applicantType } = this.props;
+    if (!applicantId || !applicantType) {
+      return null;
+    }
+    const shortInfo = firestore.data[applicantCollections[applicantType].shortInfo][applicantId];
```

Comment:
hmm that's a lot of nested square brackets, i'd suggest breaking this down into multiple lines.